src/admin/components/cell/center-cell.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { TableCell } from '@material-ui/core'
+
+import { useStyles } from 'admin/components/cell/style'
+
+export const CenterCell = ({ children }) => <TableCell align={'center'} classes={useStyles()}>
```

Comment:
This component and the `EmailCell` component are really similar. Let's create a single `Cell` component which takes its alignment as a prop.

Reply:
done